modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -432,6 +434,21 @@ public long getUserId() {
 		return _userId;
 	}
 
+	public HashMap<String, Object> getXLSXProps() {
```

Comment:
I see we're using `xlsx` but in  most other places we're using just `xls`. Can we unify?